plugin/minimal/minimal_test.go

```diff
@@ -0,0 +1,120 @@
+package minimal
+
+import (
+	"testing"
+
+	"github.com/coredns/coredns/plugin/test"
+	"github.com/miekg/dns"
+)
+
+type responseStruct struct {
+	answer []dns.RR
+	ns     []dns.RR
+	extra  []dns.RR
+	rcode  int
+}
+
+func TestMinimizeResponse(t *testing.T) {
+	baseAnswer := []dns.RR{
+		test.A("google.com.  293 IN A 142.250.76.46"),
```

Comment:
please use example.org (or example.com/net) in tests

Reply:
done